src/components/StorageOverview/Inventory/fixtures/Inventory.fixture.js

```diff
@@ -1,15 +1,16 @@
 import { Inventory } from '../Inventory';
-import { nodes, pvcs, pods, vms, vmis, migrations } from '../../fixtures/ClusterOverview.fixture';
+import { nodes, pvcs, pvs, pods, vms, cephCluster, migrations } from '../../fixtures/StorageOverview.fixture';
 
 export default [
   {
     component: Inventory,
     props: {
       nodes,
       pvcs,
+      pvs,
       pods,
       vms,
-      vmis,
```

Comment:
If vmis are not required, it should be removed from `InventoryBody.defaultProps` and `InventoryBody.propTypes` as well.

Reply:
ack